hw/ip/prim/util/dump

```diff
@@ -0,0 +1,300 @@
+8'h8d,
```

Comment:
is this file supposed to be here...? Or is this like a debug output from running the util?

Reply:
ah sorry this should not have made into this PR. I'll remove it.